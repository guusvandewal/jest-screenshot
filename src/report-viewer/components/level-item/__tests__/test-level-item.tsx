import * as React from "react";
import { shallow } from "enzyme";
import { LevelItem } from "..";

describe("LevelItem", () => {
    it("looks as expected", () => {
        expect(shallow(
            <LevelItem
                name="Some name"
                value="Some value"
            />,
        )).toMatchSnapshot();
    });
});
