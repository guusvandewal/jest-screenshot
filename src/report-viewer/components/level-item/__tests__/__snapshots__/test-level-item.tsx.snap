// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LevelItem looks as expected 1`] = `
<div
  className="level-item has-text-centered"
>
  <div>
    <p
      className="heading"
    >
      Some name
    </p>
    <p
      className="title"
    >
      Some value
    </p>
  </div>
</div>
`;
